backend/capellacollab/alembic/versions/d888f5c30aab_add_model.py

```diff
@@ -101,7 +104,7 @@ def upgrade():
             slug = base_slug
             index = 0
 
-            while slug in existing_slugs:
+            while slug in existing_slugs[project_id]:
```

Comment:
What is this doing? A comment is desired.

Reply:
It is the migration of the old databases. Per project, the slug of the model has to be unique. This was not a constraint in the old version, therefore there are conflicts (and we need to assign new slugs, adding just `-\d` to the existing "slugs").